doc/user/clusterwide.md

```diff
@@ -0,0 +1,12 @@
+# Manage clusters in all namespaces
+
+Default etcd operator behavior is to only manage etcd clusters created in the same namespace.
+It is possible to deploy an etcd operator with special option to manage clusterwide etcd clusters.
+
+## Install etcd operator
+
+etcd operator have to run with `-cluster-wide` arg option.
+
+More information in [install guide](doc/user/install_guide.md).
+
+See the example in [example/example-etcd-cluster.yaml](example/example-etcd-cluster.yaml)
```

Comment:
the [example/example-etcd-cluster.yaml](example/example-etcd-cluster.yaml) doesn't work. you might want to try [example/example-etcd-cluster.yaml](../../blob/master/example/example-etcd-cluster.yaml)

Reply:
Right. Should be 
```
[example/example-etcd-cluster.yaml](../../example/example-etcd-cluster.yaml)
```

Reply:
@hongchaodeng this `[example/example-etcd-cluster.yaml](../../example/example-etcd-cluster.yaml)` didn't work for me.

Reply:
I'm trying with `../../`

Reply:
yeah, `../../` is the right approach.